### ISSUE TITLE
Lumen 5.2 Issue.

### ISSUE THREAD

**Issue by irazasyed** [vinkla/hashids#17]
There's some problem when using with Lumen 5.2, here's the error it's throwing:

``` bash
[ErrorException]
  Argument 1 passed to Vinkla\Hashids\HashidsServiceProvider::registerFactory() must be an instance of Illuminate\Contracts\Foundation\Application, i
  nstance of Laravel\Lumen\Application given, called in /Users/User/Projects/Code/lumen/vendor/vinkla/hashids/src/HashidsServiceProvider.php on line 6
  4 and defined
```

**Comment by vinkla** [comment]
Yeah, thanks for reporting. I know about the issue though, this bug currently exists in all my packages. I'll find a better solution soon.

**Comment by irazasyed** [comment]
Fix: `use Illuminate\Contracts\Container\Container as Application;`

Same thing with another package as well. That's what fixed it. Works for both Lumen and Laravel too.

**Comment by vinkla** [comment]
Yeah, sounds like a good idea.

**Comment by irazasyed** [comment]
I'm still waiting for this package to be updated :)

I can send a PR if you'd like.

**Comment by vinkla** [comment]
I'll get to this asap.

**Comment by irazasyed** [comment]
Great. Sounds good. Thanks!

**Comment by vinkla** [comment]
I've added an implementation in https://github.com/lucadegasperi/oauth2-server-laravel/pull/646 If everything runs smoothly on that one I'll just do the same on all packages.

**Comment by irazasyed** [comment]
:+1:

**Comment by vinkla** [comment]
Fixed in 2.2.1

**Comment by irazasyed** [comment]
Thank you :)
